Makefile

```diff
@@ -37,6 +40,12 @@ pypi_package:
 install:
 	pip install -e .[notebook]
 
+install-spark-ci:
+	sudo apt-get -y install openjdk-8-jdk
+	curl https://archive.apache.org/dist/spark/spark-${SPARK_VERSION}/spark-${SPARK_VERSION}-bin-hadoop${HADOOP_VERSION}.tgz \
+	--output ${TRAVIS_BUILD_DIR}/spark.tgz
+	tar -xvzf ${TRAVIS_BUILD_DIR}/spark.tgz && mv spark-${SPARK_VERSION}-bin-hadoop${HADOOP_VERSION} spark
```

Comment:
The last "spark" isn't in the ${TRAVIS_BUILD_DIR}, right? Perhaps ${SPARK_HOME} would be better here (had some issues with this at another project).

Reply:
replaced with SPARK_DIRECTORY variable for clearer naming